plugins/inputs/vsphere/README.md

```diff
@@ -11,6 +11,8 @@ vCenter servers.
 
 ## Supported versions of vSphere
 
+This plugin supports vSphere version 5.5 through 7.0.
+
```

Comment:
Given the sentence directly below this one I do not think we should take this, right?